src/network/taraxa_capability.cpp

```diff
@@ -201,38 +206,48 @@ bool TaraxaCapability::interpretCapabilityPacketImpl(NodeID const &_nodeID, unsi
+
+            LOG(log_dg_) << "Received status message from " << _nodeID << ", peer DAG max level " << peer->dag_level_
+                         << ", peer pbft chain size " << peer->pbft_chain_size_ << ", peer syncing " << peer->syncing_
+                         << ", peer pbft round " << peer->pbft_round_ << ", peer pbft previous round next votes size "
+                         << peer->pbft_previous_round_next_votes_size_;
           }
-          LOG(log_dg_dag_sync_) << "Received status message from " << _nodeID << " DAG level:" << peer_level;
-          LOG(log_dg_pbft_sync_) << "Received status message from " << _nodeID
-                                 << " PBFT chain size:" << peer_pbft_chain_size << " " << peer->syncing_;
-          LOG(log_dg_next_votes_sync_) << "Received status message from " << _nodeID
-                                       << " PBFT round: " << peer_pbft_round;
-          peer->pbft_chain_size_ = peer_pbft_chain_size;
-          peer->dag_level_ = peer_level;
-          peer->pbft_round_ = peer_pbft_round;
-          LOG(log_dg_pbft_sync_) << "peer_pbft_chain_size: " << peer_pbft_chain_size
-                                 << ", peer_syncing_pbft_chain_size_: " << peer_syncing_pbft_chain_size_;
+
+          LOG(log_dg_dag_sync_) << "Received status message from " << _nodeID
+                                << " peer DAG max level:" << peer->dag_level_;
+          LOG(log_dg_pbft_sync_) << "Received status message from " << _nodeID << ", peer sycning: " << peer->syncing_
+                                 << ", PBFT chain size:" << peer->pbft_chain_size_
+                                 << ". Own peer syncing pbft chain size: " << peer_syncing_pbft_chain_size_;
+          LOG(log_dg_next_votes_sync_) << "Received status message from " << _nodeID << ", PBFT round "
+                                       << peer->pbft_round_ << ", peer PBFT previous round next votes size "
+                                       << peer->pbft_previous_round_next_votes_size_;
+
           if (peer->syncing_) {
-            LOG(log_dg_pbft_sync_) << "Other node is behind, prevent gossiping " << _nodeID
-                                   << "Our pbft chain size: " << pbft_chain_size
-                                   << " Peer pbft chain size: " << peer_pbft_chain_size;
-            if (syncing_ && peer_syncing_pbft == _nodeID) {
-              // We are currently syncing to a node that just reported it is
-              // not synced, force a switch to a new node
+            LOG(log_dg_pbft_sync_) << "Peer node has a short PBFT chain, prevent gossiping to " << _nodeID
+                                   << ". Own pbft chain size " << pbft_chain_size << " Peer pbft chain size "
+                                   << peer->pbft_chain_size_;
+            if (syncing_ && peer_syncing_pbft_ == _nodeID) {
+              // We are currently syncing to a node that just reported it is not synced, force a switch to a new node
```

Comment:
Could instead a log here saying we are restarting syncing because currently syncing from node that is itself syncing.

Reply:
OK